Trippy/View Controllers/Itinerary Tab/ChooseRouteViewController.m

```diff
@@ -0,0 +1,53 @@
+//
+//  ChooseRouteViewController.m
+//  Trippy
+//
+//  Created by Catherine Lu on 7/20/22.
+//
+
+#import "ChooseRouteViewController.h"
+#import "RouteCell.h"
+
+@interface ChooseRouteViewController () <UITableViewDelegate, UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UITableView *routesTableView;
+@property (weak, nonatomic) IBOutlet UIButton *selectButton;
+@property (strong, nonatomic) RouteOption *selectedRoute;
+@end
+
+@implementation ChooseRouteViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.routesTableView.delegate = self;
+    self.routesTableView.dataSource = self;
+    self.routesTableView.rowHeight = 75;
```

Comment:
nit: consider constifying this magic number. It does only show up here, which is fine.